packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -3960,6 +3960,65 @@ void main() {
         tester.getRect(find.byKey(contentKey)).left + horizontalOffset,
       );
     });
+
+    group('The menu is attached to the bottom of the MenuAnchor content', () {
+      const double contentHeight = 100.0;
+      final MenuController menuController = MenuController();
+      final UniqueKey contentKey = UniqueKey();
+
+      Finder findMenuPanel() {
+        return find.byWidgetPredicate((Widget w) => '${w.runtimeType}' == '_MenuPanel');
+      }
+
+      MenuAnchor buildMenuAnchor() {
+        return MenuAnchor(
+          controller: menuController,
+          builderAlignment: AlignmentDirectional.topStart,
+          menuChildren: <Widget>[
+            MenuItemButton(
+              onPressed: () {},
+              child: const Text('Item 1'),
+            ),
+          ],
+          builder: (BuildContext context, MenuController controller, Widget? child) {
+            return SizedBox(key: contentKey, width: 100.0, height: contentHeight);
+          },
+          child: Container(width: 200, height: 60, color: Colors.purple),
+        );
+      }
+
+      testWidgets('when given loose constraints', (WidgetTester tester) async {
+        await tester.pumpWidget(MaterialApp(
```

Comment:
Turbo nit: I usually add a newline after pumpWidget 
e.g.
```dart 
await tester.pumpWidget(
  MaterialApp(
   ...
  )
)
```
But that's opinionated.

Reply:
Most of the existing tests I worked on when I started to contribute, do not add a new line but several others do.
In this test file, a newline is added so I fully agree that it will be better.
Fortunately there is a good chance that we will have autoformat in 2025.